sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormSelectionMark.cs

```diff
@@ -28,7 +28,7 @@ internal FormSelectionMark(FieldBoundingBox boundingBox, int pageNumber, string
         public float Confidence { get; }
 
         /// <summary>
-        /// Selection mark state value.
+        /// Selection mark state value, like Selected or Unselected.
         /// </summary>
         public SelectionMarkState State { get; }
```

Comment:
`like Selected or Unselected` makes it sound like there are other possible states (are there?). It may be worth rephrasing it a bit to make it clear that these are the only states.

Reply:
For now there aren't, but in the future `Undefined` might be included